ip-bom/pom.xml

```diff
@@ -1266,6 +1266,18 @@
       </dependency>
 
       <dependency>
+        <groupId>org.apache.pdfbox</groupId>
+        <artifactId>pdfbox</artifactId>
+        <version>${version.org.apache.pdfbox}</version>
+        <exclusions>
+          <exclusion>
+            <groupId>commons-logging</groupId>
```

Comment:
We do rely on SLF4J only for logging, so we need to exclude every logging implementation the dependencies may be dragging in. Ideally (if Maven would support this), we would add an `<inclusion>` replacing the `commons-logging` with `org.slf4j.:jcl-over-slf4`. Since Maven does not support that, one needs to add it manually on all the places the pdfbox is used. I know this is far from ideal, but it's the best option we found if we want to support unified logging.

That being said, we can do that it our BOMs if this is not something we all agree on.
SLF4J pages explain this in more detail: http://www.slf4j.org/legacy.html